pkg/asset/manifests/ingress_test.go

```diff
@@ -91,6 +93,15 @@ func TestGenerateIngerssDefaultPlacement(t *testing.T) {
 			infrastructureTopology:    configv1.HighlyAvailableTopologyMode,
 			expectedIngressPlacement:  configv1.DefaultPlacementWorkers,
 		},
+		{
+			name:                        "test setting of aws lb type",
+			installConfigBuildOptions:   []icOption{icBuild.withLBType(configv1.NLB)},
+			controlPlaneTopology:        configv1.HighlyAvailableTopologyMode,
+			infrastructureTopology:      configv1.HighlyAvailableTopologyMode,
+			expectedIngressPlacement:    configv1.DefaultPlacementWorkers,
+			expectedIngressAWSLBType:    configv1.NLB,
+			expectedIngressPlatformType: configv1.AWSPlatformType,
+		},
```

Comment:
I think "vanilla aws" should be augmented to check if the expectedIngressAWSLBType and expectedIngressPlatformType continue to behave as expected for the default "Classic" case after these changes.

Reply:
Actually that is not needed as for Classic LB setting LBType is not mandatory. The reason is LBType field comes under ProviderParameters which is not mandatory for Classic.

Reply:
I understand that it is not mandatory. Shouldn't we have a test for all input values? It appears that we have one when it is not set, "NLB" but not one for when it is explicitly set to "Classic".

Reply:
Yes have added the test case for it as well. Please check line 106.